src/main/java/com/minecolonies/coremod/util/RecipeHandler.java

```diff
@@ -137,9 +139,137 @@ public static void init(final boolean enableInDevelopmentFeatures, final boolean
+                new ItemStack(ModBlocks.blockTimberFrameOak, FOUR_BLOCKS, 4 ), "SSS", "SXS", "SBS", 'S', WOODEN_STICK, 'X', new ItemStack(Blocks.PLANKS, 1, 0), 'B', ModItems.buildTool));
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.blockTimberFrameOak, FOUR_BLOCKS, 5), "X  ", " X ", " BX", 'X', new ItemStack(Blocks.PLANKS, 1, 0), 'B', ModItems.buildTool));
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.blockTimberFrameOak, FOUR_BLOCKS, 6), "  X", " X ", "XB ", 'X', new ItemStack(Blocks.PLANKS, 1, 0), 'B', ModItems.buildTool));
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.blockTimberFrameOak, FOUR_BLOCKS, 7), "SXS", "SBS", "SSS", 'S', WOODEN_STICK, 'X', new ItemStack(Blocks.PLANKS, 1, 0), 'B', ModItems.buildTool));
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.blockTimberFrameOak, FOUR_BLOCKS, 8),"SSS", "XXX", "SBS", 'S', WOODEN_STICK, 'X', new ItemStack(Blocks.PLANKS, 1, 0), 'B', ModItems.buildTool));
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.blockTimberFrameOak, FOUR_BLOCKS, 9), "   ", "XXX", " B ", 'X', new ItemStack(Blocks.PLANKS, 1, 0), 'B', ModItems.buildTool));
+
+
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.blockTimberFrameSpruce, FOUR_BLOCKS, 0), " X ", " X ", " XB", 'X', new ItemStack(Blocks.PLANKS, 1, 1), 'B', ModItems.buildTool));
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.blockTimberFrameSpruce, FOUR_BLOCKS, 1), "X X", " X ", "XBX", 'X', new ItemStack(Blocks.PLANKS, 1, 1), 'B', ModItems.buildTool));
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.blockTimberFrameSpruce, FOUR_BLOCKS, 2), "SXS", "XBX", "SXS", 'S', WOODEN_STICK, 'X', new ItemStack(Blocks.PLANKS, 1, 1), 'B', ModItems.buildTool));
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.blockTimberFrameSpruce, FOUR_BLOCKS, 3), "SXS", "SXS", "SBS", 'S', WOODEN_STICK, 'X', new ItemStack(Blocks.PLANKS, 1, 1), 'B', ModItems.buildTool));
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.blockTimberFrameSpruce, FOUR_BLOCKS, 4 ), "SSS", "SXS", "SBS", 'S', WOODEN_STICK, 'X', new ItemStack(Blocks.PLANKS, 1, 1), 'B', ModItems.buildTool));
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.blockTimberFrameSpruce, FOUR_BLOCKS, 5), "X  ", " X ", " BX", 'X', new ItemStack(Blocks.PLANKS, 1, 1), 'B', ModItems.buildTool));
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.blockTimberFrameSpruce, FOUR_BLOCKS, 6), "  X", " X ", "XB ", 'X', new ItemStack(Blocks.PLANKS, 1, 1), 'B', ModItems.buildTool));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 6 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)